### ISSUE TITLE
mongodb error on deploy eosio.system

### ISSUE THREAD

**Issue by kesar** [EOSIO/eos#1820]
```
2379505ms thread-0   chain_controller.cpp:418      _apply_cycle_trace   ] [(eosio,issue)->eosio]: CONSOLE OUTPUT BEGIN =====================
2379505ms thread-0   chain_controller.cpp:419      _apply_cycle_trace   ] eosio::issue
2379505ms thread-0   chain_controller.cpp:420      _apply_cycle_trace   ] [(eosio,issue)->eosio]: CONSOLE OUTPUT END   =====================
2379505ms thread-0   chain_controller.cpp:418      _apply_cycle_trace   ] [(eosio,transfer)->eosio]: CONSOLE OUTPUT BEGIN =====================
2379505ms thread-0   chain_controller.cpp:419      _apply_cycle_trace   ] eosio::transfer
2379505ms thread-0   chain_controller.cpp:420      _apply_cycle_trace   ] [(eosio,transfer)->eosio]: CONSOLE OUTPUT END   =====================
eosio generated block 8e9e1e7a... #124 @ 2018-03-25T12:39:39.500 with 2 trxs
2379519ms thread-1   mongo_db_plugin.cpp:266       add_data             ] Unable to convert action.data to ABI: eosio :: issue, what: Assert Exception (10)
itr != structs.end(): Unknown struct 

2379519ms thread-1   mongo_db_plugin.cpp:266       add_data             ] Unable to convert action.data to ABI: eosio :: transfer, what: Assert Exception (10)
itr != structs.end(): Unknown struct 

2379527ms thread-1   mongo_db_plugin.cpp:266       add_data             ] Unable to convert action.data to ABI: eosio :: issue, what: Assert Exception (10)
itr != structs.end(): Unknown struct 

eosio generated block 1980ce2d... #125 @ 2018-03-25T12:39:40.000 with 0 trxs
2380017ms thread-1   mongo_db_plugin.cpp:634       update_account       ] from_balance: 0.0000 EOS to_balance: 0.0000 EOS asset_quantity: 10000.0000 EOS 
``` 

Running:
```
nodeos --genesis-json genesis.json --config-dir . --data-dir ./data --enable-stale-production --replay-blockchain

cleos set contract eosio /eos/build/contracts/eosio.system/eosio.system.wast /eos/build/contracts/eosio.system/eosio.system.abi

cleos push action eosio issue '{"to":"eosio", "quantity":"10000.0000 EOS" }' -p eosio@active
```

**Comment by kesar** [comment]
<img width="1047" alt="screen shot 2018-03-25 at 14 49 34" src="https://user-images.githubusercontent.com/1288106/37875172-db3161a0-303b-11e8-887d-7a291ed4cb6d.png">

Also seems to be duplicated

**Comment by andriantolie** [comment]
I think mongodb plugin is outdated by the time this issue is created. It's fixed in PR https://github.com/EOSIO/eos/pull/4304 and planned to be released on v1.1. I'm going to close this issue, please try again with v1.1 later and if you still encounter the problem please report it as new issue. Thanks!
